lib/maze/configuration.rb

```diff
@@ -9,6 +9,7 @@ def initialize
       self.receive_no_requests_wait = 30
       self.receive_requests_wait = 30
       self.enforce_bugsnag_integrity = true
+      self.captured_invalid_requests = [:errors, :sessions, :builds, :uploads]
```

Comment:
I'd make this a set since it doesn't make sense to have duplicates and `includes?` is faster than on an array (almost definitely makes 0 difference when there are so few elements though)

```suggestion
      self.captured_invalid_requests = Set[:errors, :sessions, :builds, :uploads]
```